docs/usage/web-api.md

```diff
@@ -227,6 +228,7 @@ may still be temporarily unavailable:
 - `https://walrus-testnet-publisher.stakely.io`
 - `https://walrus-testnet-publisher.criterionvc.com`
 - `http://37.27.230.228:9001`
+- `https://walrus-testnet-published.luckyresearch.org`
```

Comment:
Question: Is this intentionally called "published" (instead of "publishe**r**")?